backbone.js

```diff
@@ -543,6 +543,18 @@
       return this.set(attr, void 0, _.extend({}, options, {unset: true}));
     },
 
+    // accepts an attribute name (or array of attribute names) and a transformer
+    // function which maps the current attribute value(s) to new attribute value(s)
+    update: function(attrName, transformer, setOptions) {
+      var attrNames = _.isArray(attrName) ? attrName : [attrName];
+      var newValues = _.reduce(attrNames, function(valueCollector, attr) {
```

Comment:
I was more thinking of monkey patching the lodash reduce to mimic underscore API. I thought Lodash compat wasn't guaranteed (it's untested and the Lodash API has widely diverged), but per @megawac's comment you must be right, so nevermind.